### PR TITLE
Add Tags To Exports

### PR DESCRIPTION
**Pull request by sampoder**
Exports are cool. Tags are cool. I was exporting some stuff from Outernet and I thought it'd be cool if Alex's tags he added were in the export.

### CODE REVIEW

File: app/services/canonical_transaction_service/export/csv.rb [hunk 1]
```diff
@@ -28,15 +28,23 @@ def event
...
 
       def row(ct)
-        ::CSV::Row.new(headers, [ct.date, ct.local_hcb_code.memo, ct.amount_cents])
+        ::CSV::Row.new(
+          headers,
+          [
+            ct.date,
+            ct.local_hcb_code.memo,
+            ct.amount_cents,
+            ct.local_hcb_code.tags.filter{ |tag| !@event || tag.event == @event }.map { |tag| tag.label }.join(", ")
```
Comment (cjdenio) [inline app/services/canonical_transaction_service/export/csv.rb]:
```suggestion
            ct.local_hcb_code.tags.filter { |tag| tag.event_id == @event_id }.pluck(:label).join(", ")
```

**Comment by sampoder** [comment]
oh cool i never knew about pluck!

File: app/services/canonical_transaction_service/export/json.rb [hunk 2]
```diff
@@ -26,7 +26,8 @@ def row(ct)
         {
           date: ct.date,
           memo: ct.local_hcb_code.memo,
-          amount_cents: ct.amount_cents
+          amount_cents: ct.amount_cents,
+          tags: ct.local_hcb_code.tags.filter{ |tag| !@event || tag.event == @event }.map { |tag| tag.label }
```
Comment (sampoder) [inline app/services/canonical_transaction_service/export/json.rb]:
```suggestion
          tags: ct.local_hcb_code.tags.filter { |tag| tag.event_id == @event_id }.pluck(:label).join(", ")
```

**Review by cjdenio — APPROVED** [review]
Thanks!

**Pull request merged**